model/src/main/java/oracle/kubernetes/weblogic/domain/v2/BaseConfiguration.java

```diff
@@ -61,28 +48,6 @@
           + "Legal values are NEVER, ALWAYS, or IF_NEEDED.")
   private String serverStartPolicy;
 
-  /**
-   * Defines the settings for the liveness probe. Any that are not specified will default to the
-   * runtime liveness probe tuning settings.
-   *
-   * @since 2.0
-   */
-  @SerializedName("livenessProbe")
-  @Expose
-  @Description("Settings for the liveness probe associated with a server")
-  private V1Probe livenessProbe = new V1Probe();
```

Comment:
This doesn't look right... The wiki only calls out that the customer will be able to modify the tuning parameters of the liveness and readiness probe; not replace the entire probe.  Customers wouldn't know what to put here.